ui/src/views/user/signup-thankyou/SignupThankyouView.vue

```diff
@@ -26,4 +26,12 @@ export default { name: "SignupThankyouView" };
 </script>
 
 <style lang="scss">
+  @media only screen and (max-width: map-get($breakpoints, "xxs")) {
```

Comment:
The base font size was already set somewhere in the mixins/components. This is a good approach, but its better to set the base body font size instead to make the font sizes proportional to each other. Tho I myself have never targeted the xxs media query so far in this code, md is enough I think

Reply:
It is set in the ContentCard.vue component for "md" and lower screen sizes. The issue is, the font size set by the component is still too large for xxs screens, as shown in my screenshot above. Hence the reason I applied the font size for this particular page. So, do you want me to keep this fix only for this particular page, or set the font size for xxs in the component, or discard it entirely?

Reply:
Please set it in the Content Card component, thank you!

Reply:
But again, I myself have never targeted other breakpoints but md XD  you are really diligent

You can also use the include-font-size mixin btw

Reply:
Hi, there is an issue with the ContentCard.vue component, at least on my machine.

<img width="466" alt="asdf" src="https://user-images.githubusercontent.com/64476430/172060709-22d68c31-0e19-4f4b-9c4a-fd64f939a78a.PNG">

The line `@include font-size(1.25);` doesn't work; I changed the value from 1.25 to 2.50 in the screenshot just to confirm if it's working, and it's not. The title size in the output page didn't change. I propose changing the line to something like `font-size: 1.25rem !important;` if it's ok

Reply:
I see, what command are u running in ur machines? `yarn serve` should work and recompile the code on each change. But if its not possible, then using font size directly is also fine

But i

Reply:
I decided not to change the original code, I just added additional media query for xxs that use `font-size: 1rem !important;` to fix the text clipping. Nevertheless, I've followed the change requests so I hope everything is ok now. Please review the files once more.

Reply:
The 404 page code looks great, thank you! Can you update the screenshots?